docs/content/querying/topnmetricspec.md

```diff
@@ -28,39 +28,26 @@ The metric field can also be given as a JSON object. The grammar for dimension v
 |type|this indicates a numeric sort|yes|
 |metric|the actual metric field in which results will be sorted by|yes|
 
-## Lexicographic TopNMetricSpec
+## Dimension TopNMetricSpec
```

Comment:
I think it'd be good to include a small note that lexicographic and alphanumeric specs are still supported for backwards compatibility, but are deprecated. Not a full section - just a note to reassure people that they won't stop working after an upgrade.